wazo_provd/tzinform.py

```diff
@@ -150,13 +150,29 @@ def get_timezone_info(self, timezone_name: str) -> TimeZoneInfoDict:
             logger.debug('Could not find zone for %s', timezone_name)
             raise TimezoneNotFoundError(timezone_name)
 
-        seconds = int(zone.std.utcoff.total_seconds())
-        transitions = zone.transitions(datetime.now().year)
-        dst_start = datetime.fromtimestamp(transitions[0])
-        dst_end = datetime.fromtimestamp(transitions[1])
-        return {
-            'utcoffset': Time(seconds),
-            'dst': DSTRuleDict(
+        if hasattr(zone, "std"):
+            seconds = int(zone.std.utcoff.total_seconds())
+        else:
+            seconds = 0
+
+        if hasattr(zone, "dst"):
+            save = int(zone.dst.dstoff.total_seconds())
+        else:
+            save = 0
+
+        if hasattr(zone, "transitions"):
+            transitions = zone.transitions(datetime.now().year)
```

Comment:
Should we use `datetime.now(timezone.utc)` to avoid corner case at the end of the year ?